scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -320,7 +318,7 @@ export class LaneRemoveCmd implements Command {
       const removePromptResult = await approveOperation();
       // @ts-ignore
       if (!yn(removePromptResult.shouldProceed)) {
-        throw new BitError('the operation has been canceled');
+        throw new BitError('the operation has been canceled'); // @david can we be more specific - i know this comes from legacy, but is there any more information we can add to this specific error point?
```

Comment:
It's just when the user answers "no" to the prompt, or when hitting Ctrl+C, so the message looks good to me, but you can change it to anything else. (you can also change the prompt itself in the legacy).

Reply:
@benjgil , this is funny, I didn't think it's a typo, so I googled it, turned out British spells it with two `l` while American with one `l` :) 

https://www.merriam-webster.com/grammar/canceled-or-cancelled#:~:text='Canceled'%20or%20'Cancelled'%3F,One%20L%20or%20two%3F&text=While%20both%20canceled%20and%20cancelled,more%20common%20in%20British%20English.